### PR TITLE
Add CondIsUnbreakable

### PR DESCRIPTION
**Pull request by Mr-Darth**
### Description
Adds a condition to check if an itemtype is unbreakable.

---
**Target Minecraft Versions:** any
**Requirements:** 1.11+
**Related Issues:** none

### CODE REVIEW

**Comment by TPGamesNL** [comment]
AFAIK `ItemMeta#isUnbreakable` was added in 1.11.x

**Comment by Mr-Darth** [comment]
> AFAIK `ItemMeta#isUnbreakable` was added in 1.11.x

Fixed, sorry 😅

**Comment by TPGamesNL** [comment]
> > AFAIK `ItemMeta#isUnbreakable` was added in 1.11.x
> 
> Fixed, sorry 😅

No worries. 2 suggestions: you can add compatibility by using `ItemFlag`s and it could be a nice addition to add a way to make an item unbreakable as well.

File: src/main/java/ch/njol/skript/conditions/CondIsUnbreakable.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/conditions/CondIsUnbreakable.java:18]: bensku has updated the last line of the license heard which can be found [**HERE**](https://github.com/SkriptLang/Skript/blob/master/licenseheader.txt) to update accordingly

File: src/main/java/ch/njol/skript/conditions/CondIsUnbreakable.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+
+@Name("Is Unbreakable")
+@Description("Checks whether an item is unbreakable.")
+@Examples("if event-item is unbreakable")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.11+")
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/conditions/CondIsUnbreakable.java:37]: To keep consistency with the rest of this project, I suggestion:
```suggestion
@RequiredPlugins("Minecraft 1.11+")
```

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
could small changes, other than that looks good to me

**Review by ShaneBeee — APPROVED** [review]

**Review by ShaneBeee — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]

**Comment by ShaneBeee** [comment]
Thank you for your contribution

**Pull request merged**